ccdaservice/utils/demographics/populate-demographics.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const { fetchPreviousAddresses } = require('./previous-addresses');
+const { fDate } = require('../date/date');
+const {
+    NOT_INFORMED,
+    NULL_FLAVOR,
+    DECLINED_TO_SPECIFY,
+} = require('../constants');
+
+function getGuardianNames(display_name) {
+    const parts = display_name.split(' ');
+    return parts.length === 3
+        ? [{ first: parts[0], last: parts[2] }]
+        : parts.length === 2
+        ? [{ first: parts[0], last: parts[1] }]
+        : [{ first: NOT_INFORMED, last: NOT_INFORMED }];
+}
+
+function getGuardianInfo(guardian) {
+    return [
+        {
+            relation: guardian.relation,
+            addresses: [
+                {
+                    street_lines: [guardian.address],
+                    city: guardian.city,
+                    state: guardian.state,
+                    zip: guardian.postalCode,
+                    country: guardian.country || 'US',
+                    use: 'primary home',
```

Comment:
I've never really liked trailing commas due to switching between JSON vs non, but I'm guessing this is the style syntax we are following?

Reply:
I'm following this style but it's totally optional, the linter doesn't enforce it. Quoting MDN:

> Trailing commas (sometimes called "final commas") can be useful when adding new elements, parameters, or properties to JavaScript code. If you want to add a new property, you can add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.